bullet/src/SDFFeatures.cc

```diff
@@ -172,6 +172,27 @@ Identity SDFFeatures::ConstructSdfCollision(
   return this->GenerateInvalidId();
 }
 
+/////////////////////////////////////////////////
+std::size_t SDFFeatures::FindOrConstructSdfLink(
```

Comment:
Is this a physics plugin function, or a custom one for the engine? Who consumes this one?

Reply:
It is part of the plugin, but not part of the default functions consumed by ignition. It's consumed internally to create links avoiding duplicates if the link is already created. I had it inside the createModel function (The one you suggested me to erase) and I plan to use it inside createJoint to get the entities of the links involved in the joint creation.